Makefile

```diff
@@ -143,10 +147,10 @@ _add_openshift_push_permissions:
 	oc adm policy add-role-to-user system:image-builder $(_OPENSHIFT_USERNAME) || true
 .PHONY: _add_openshift_push_permissions
 
-push-image-to-local-openshift: _add_openshift_push_permissions _wait_for_local_docker_registry _login_to_docker push-image-common
+push-image-to-local-openshift: _add_openshift_push_permissions _wait_for_local_docker_registry _login_to_openshift _login_to_docker push-image-common
 .PHONY: push-image-to-local-openshift
 
-push-image-to-online-openshift: _login_to_docker push-image-common
+push-image-to-online-openshift: _login_to_openshift _login_to_docker push-image-common
```

Comment:
With OpenShift online I'm already logged in at this point. And I don't even know the password. I'm given a token.

Reply:
Ok, I'll remove this